### ISSUE TITLE
Warnings about files being moved

### ISSUE THREAD

**Issue by geelen** [styled-components/styled-components#918]
Is anyone else getting this? Running `yarn test` gives you a bunch of output like this:

```
 PASS  src/models/test/ThemeProvider.test.js
  ● Console

    console.error node_modules/fbjs/lib/warning.js:36
      Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: Shallow renderer has been moved to react-test-renderer/shallow. Update references to remove this warning.
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: Failed context type: Invalid context `__styled-components__` of type `function` supplied to `Child`, expected `object`.
          in Child
          in ThemeProvider
          in ThemeProvider

 PASS  src/constructors/test/injectGlobal.test.js
  ● Console

    console.error node_modules/fbjs/lib/warning.js:36
      Warning: ReactTestUtils has been moved to react-dom/test-utils. Update references to remove this warning.
    console.error node_modules/fbjs/lib/warning.js:36
      Warning: Shallow renderer has been moved to react-test-renderer/shallow. Update references to remove this warning.
```

Do we need to bump some versions or something? Do we have hard-coded paths somewhere? Is it just broken on my machine?

**Comment by k15a** [comment]
These got moved with React v15.5.0. Maybe we just need to update some dependencies and replace all references to `react-addons-test-utils` with `react-dom/test-utils` and `react-test-renderer/shallow`

[react-test-utils](https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#react-test-utils), [react-test-renderer](https://facebook.github.io/react/blog/2017/04/07/react-v15.5.0.html#react-test-renderer).

**Comment by mxstbr** [comment]
This was fixed at some point, can't replicate locally anymore. Closing!
